tests/behave/features/search_legal_advisers.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Search for Education and Discrimination categories
```

Comment:
:raising_hand_man: This title should reflect the filename: `search_legal_advisers.feature`